uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/screens/habits/list/HabitCardListCache.kt

```diff
@@ -63,6 +64,8 @@ class HabitCardListCache @Inject constructor(
     private var filteredHabits: HabitList
     private val taskRunner: TaskRunner
 
+    private val logger = StandardLogger("HabitCardListCache")
```

Comment:
`AndroidLogger` seems to be in `uhabits-android`, while this code is in `uhabits-core`, should I move `AndroidLogger` there?

Reply:
Sorry, I didn't notice that. In this case, you can just add `logging: Logging` to the constructor, then call `logging.getLogger("HabitCardListCache")`. Dagger will include the correct one according to the platform (Android gets `AndroidLogger`, desktop gets `StandardLogger`, etc).

```kotlin
@AppScope
class HabitCardListCache @Inject constructor(
    private val allHabits: HabitList,
    private val commandRunner: CommandRunner,
    taskRunner: TaskRunner,
    logging: Logging,
) : CommandRunner.Listener {
    private val logger = logging.getLogger("HabitCardListCache")
```

Notice also that we can't move `AndroidLogger` to `uhabits-core` because it depends on `android.util.Log`.